SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -9,8 +9,10 @@ import SwiftUI
 import ReadabilityModifier
 
 struct SponsorsView: View {
-    @Environment(\.openURL) var openURL
+    @Environment(\.openURL) private var openURL
     @StateObject private var viewModel = SponsorsViewModel()
+    @State private var isPresentWebView = false
+    @State private var urlToOpen = ""
```

Comment:
nit: We do not need both an isPresentWebView and a urlToOpen.

Utilising the item initialiser (https://developer.apple.com/documentation/swiftui/view/sheet(item:ondismiss:content:)) we can have an optional URL or String which will automatically show the sheet when it's defined (and is set to nil the rest of the time).